google/resumable_media/requests/__init__.py

```diff
@@ -659,6 +659,18 @@ def SimpleUpload(*args, **kwargs):
    True
 """
 
+import pkg_resources
+
+try:
+    pkg_resources.require('requests >= 2.18.0')
```

Comment:
@jonparrott Should I make this less strict / less new?

I only need `requests.models.Response` to be a context manager and `urllib3` to be a separate dependency rather than vendored/bundled in.

Reply:
I don't feel strongly.

Reply:
Cool, we shall let it be then unless we hear otherwise.